html/pfappserver/lib/pfappserver/Form/Config/WMI.pm

```diff
@@ -0,0 +1,96 @@
+package pfappserver::Form::Config::WMI;
+
+=head1 NAME
+
+pfappserver::Form::Config::WMI - Web form for a WMI
+
+=head1 DESCRIPTION
+
+Form definition to create or update WMI.
+
+=cut
+
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Base::Form';
+with 'pfappserver::Base::Form::Role::Help';
+
+use pf::config;
+use pf::util;
+
+## Definition
+has_field 'id' =>
+  (
+   type => 'Text',
+   label => 'Name',
+   required => 1,
+   messages => { required => 'Please specify a name for the scan engine' },
+  );
+
+has_field 'request' =>
+  (
+   type => 'Text',
+   label => 'Request',
+   required => 1,
+   messages => { required => 'Please specify a Request' },
```

Comment:
We have (basic) help built-in in these config modules. Specifying a help attribute here would show the text under it. I guess it would be better than nothing ?